src/aggregator/src/test/java/au/org/emii/aggregator/NetcdfAggregatorTest.java

```diff
@@ -280,6 +281,26 @@ public void testCarsMonthly() throws IOException, AggregationException {
         assertNetcdfFileEqualsCdl(resourcePath("au/org/emii/aggregator/cars-monthly-expected.cdl"), outputFile);
     }
 
+    @Test
+    public void testCarsDepth() throws IOException, AggregationException {
+        LatLonRect bbox = new LatLonRect(new LatLonPointImmutable(-44.5, 114), new LatLonPointImmutable(-44.5, 114));
+
+        CalendarDateRange timeRange = CalendarDateRange.of(CalendarDate.parseISOformat("gregorian", "2009-01-01T00:00:00"),
+                CalendarDate.parseISOformat("gregorian", "2009-12-26T00:00:00"));
```

Comment:
Maybe it's not worth trying to subset on time since what we're testing here is the depth subset feature.